### PR TITLE
fix(card-and-account-new): update profile id for insturments accordin…

### PR DESCRIPTION
**Pull request by gaucj-t**
Since PUK was introduced we need to utilise the correct profile id according to the payment jurisdiction.

### CODE REVIEW

**Review by oleggrech-t — APPROVED** [review]

**Review by jasnvella2 — APPROVED** [review]

File: mixins/BaseMixin.ts [hunk 1]
```diff
@@ -45,6 +41,38 @@ export default class BaseMixin extends Vue {
             : this.$config.profileId.managed_cards_corporates!
     }
 
+    get identityRegCountryIsUK() {
+        return this.isConsumer ? this.stores.consumers.isUk : this.stores.corporates.isUk
+    }
+
+    get cardJuristicationProfileId() {
```
Comment (bartolo-maureen) [inline mixins/BaseMixin.ts:48]: Very minor but I think you have a typo in Jurisdiction

File: mixins/BaseMixin.ts [hunk 2]
```diff
@@ -45,6 +41,38 @@ export default class BaseMixin extends Vue {
...
+            return this.$config.profileId.managed_cards_consumers
+        } else {
+            if (this.identityRegCountryIsUK) {
+                return this.$config.profileId.managed_cards_corporates_uk
+            }
+            return this.$config.profileId.managed_cards_corporates
+        }
+    }
+
+    get accountJuristicationProfileId() {
```
Comment (bartolo-maureen) [inline mixins/BaseMixin.ts:62]: Same here

File: mixins/BaseMixin.ts [hunk 3]
```diff
@@ -70,15 +98,19 @@ export default class BaseMixin extends Vue {
...
+        }
     }
 
     get rootSurname(): string {
         if (this.isConsumerPopulated) {
             return this.stores.consumers.consumer!.rootUser.surname
         } else if (this.isCorporatePopulated) {
             return this.stores.corporates.corporate!.rootUser.surname
-        } else return 'nosurname'
+        } else {
```
Comment (PeterThrels) [inline mixins/BaseMixin.ts:111]: no need for last else, just return

**Review by PeterThrels — APPROVED** [review]

File: mixins/BaseMixin.ts (hunk 1 above)
Comment (gaucj-t) [reply mixins/BaseMixin.ts]: sorted. nice one

File: mixins/BaseMixin.ts (hunk 3 above)
Comment (gaucj-t) [reply mixins/BaseMixin.ts]: sorted also

**Review by bartolo-maureen — APPROVED** [review]

**Pull request merged**